Tests/Functional/Doctrine/Orm/PageTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2014 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\SimpleCmsBundle\Tests\Functional\Doctrine\Orm;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\ORM\EntityManager;
+use Sonata\CoreBundle;
+use Symfony\Cmf\Component\Testing\Functional\BaseTestCase;
+use Symfony\Cmf\Bundle\SimpleCmsBundle\Doctrine\Orm\Page;
+
+class PageTest extends BaseTestCase
+{
+    /**
+     * @var EntityManager
+     */
+    private $em;
+
+    /**
+     * @var Page
+     */
+    private $basePage;
+
+    public function setUp()
+    {
+        $this->em = $this->db('ORM')->getOm();
+
+        $page = new Page(array('add_locale_pattern' => true));
+        $page->setName('base-page-name');
+        $page->setTitle('Base Page Title');
```

Comment:
we should add orm (and in the phpcr fixtures phpcr) everywhere and make the tests check if they see the expected entity/document or always phpcr.